### DISCUSSION TITLE
RTL table issue

### DISCUSSION THREAD

**Discussion by rzasharifi** [spacecowboy/Feeder#306] [General]
Hi deves,
As you can see, rtl table which is written in Persian language is seen mirrored in the app! In addition, you should drag the table to see all the contents. Would you mind take a look at it? 
Thanks in advance 
[The news source](https://gadgetnews.net/875911/loan-to-buy-this-car-instead-peugeot-pars/)

**Comment by rzasharifi** [comment]
![IMG_20240613_155542](https://github.com/spacecowboy/Feeder/assets/64254187/9b8ad125-504f-43f4-8e4a-cf79b2cc9ecb)
![IMG_20240613_155521](https://github.com/spacecowboy/Feeder/assets/64254187/c639c8e4-4e7f-46cf-b13c-761f14c757bf)

**Comment by spacecowboy** [comment]
Note to self, RTL is defined in top level html tag with attribute 'dir="rtl"'

**Comment by spacecowboy** [comment]
Fixed in https://github.com/spacecowboy/Feeder/pull/312

**Reply by rzasharifi** [reply]
![Screenshot_2024-06-18-22-04-33-030_org mozilla fennec_fdroid](https://github.com/spacecowboy/Feeder/assets/64254187/e5ccf36c-014c-4949-91dd-581cfb7f50ab)
![Screenshot_2024-06-18-22-14-00-355_com nononsenseapps feeder-edit](https://github.com/spacecowboy/Feeder/assets/64254187/024d09f0-56da-4398-97e7-558e8db6d877)
![IMG_20240618_221213](https://github.com/spacecowboy/Feeder/assets/64254187/60cc99a8-83a2-4b44-9c45-85c4c1a3e6e3)
![IMG_20240618_221148](https://github.com/spacecowboy/Feeder/assets/64254187/bff068c2-0a4e-42ae-8907-8b23a9d6348b)

Hi again. Thank you for hard working. I am afraid there are some problems! In mixed languages titles there is an issue. In addition, tables in RTL languages should be in order as they are written from right to the left.
